# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxFontEntry.h
# Commit: e6523049b675
# Full Hash: e6523049b6751146641d019b9adc0825473c46aa
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 09:10:29
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 1 - Move COLR font support functions from gfxUtils into their own class. r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152036
# ==============================================================================

diff -r 90f3275f3489 -r e6523049b675 gfx/thebes/gfxFontEntry.h
--- a/gfx/thebes/gfxFontEntry.h	Tue Aug 16 12:40:10 2022 +0000
+++ b/gfx/thebes/gfxFontEntry.h	Tue Aug 16 12:48:31 2022 +0000
@@ -9,6 +9,7 @@
 #include <math.h>
 #include <new>
 #include <utility>
+#include "COLRFonts.h"
 #include "ThebesRLBoxTypes.h"
 #include "gfxFontUtils.h"
 #include "gfxFontVariations.h"
@@ -274,7 +275,7 @@
                           nsTArray<mozilla::gfx::DeviceColor>& layerColors);
   bool HasColorLayersForGlyph(uint32_t aGlyphId) {
     MOZ_ASSERT(GetCOLR());
-    return gfxFontUtils::HasColorLayersForGlyph(GetCOLR(), aGlyphId);
+    return mozilla::gfx::COLRFonts::HasColorLayersForGlyph(GetCOLR(), aGlyphId);
   }
 
   bool HasColorBitmapTable() {